addons/mail/static/tests/discuss_app/discuss_tests.js

```diff
@@ -48,62 +49,79 @@ QUnit.test("sanity check", async (assert) => {
     ]);
 });
 
-QUnit.test("can change the thread name of #general", async (assert) => {
+QUnit.test("can change the thread name of #general [REQUIRE FOCUS]", async (assert) => {
     const pyEnv = await startServer();
     const channelId = pyEnv["discuss.channel"].create({
         name: "general",
         channel_type: "channel",
+        create_uid: pyEnv.currentPartnerId,
     });
+    const def = new Deferred();
     const { openDiscuss } = await start({
         mockRPC(route, params) {
             if (route === "/web/dataset/call_kw/discuss.channel/channel_rename") {
                 assert.step(route);
+                def.resolve();
             }
         },
     });
-    await openDiscuss(channelId);
-    assert.containsOnce($, "input.o-mail-Discuss-threadName");
-    const $name = $("input.o-mail-Discuss-threadName");
-
-    click($name).catch(() => {});
-    assert.strictEqual($name.val(), "general");
-    await editInput(document.body, "input.o-mail-Discuss-threadName", "special");
-    await triggerEvent(document.body, "input.o-mail-Discuss-threadName", "keydown", {
-        key: "Enter",
-    });
-    assert.strictEqual($name.val(), "special");
+    openDiscuss(channelId);
+    await contains(".o-mail-Composer-input:focus");
+    await contains("input.o-mail-Discuss-threadName", { value: "general" });
+    await webInsertText("input.o-mail-Discuss-threadName", "special", { replace: true });
+    triggerHotkey("Enter");
+    await contains(".o-mail-DiscussCategoryItem", { text: "special" });
+    await contains("input.o-mail-Discuss-threadName", { value: "special" });
+    await def;
     assert.verifySteps(["/web/dataset/call_kw/discuss.channel/channel_rename"]);
 });
 
-QUnit.test("can change the thread description of #general", async (assert) => {
+QUnit.test("can active change thread from messaging menu", async (assert) => {
+    const pyEnv = await startServer();
+    const [, teamId] = pyEnv["discuss.channel"].create([
+        { name: "general", channel_type: "channel" },
+        { name: "team", channel_type: "channel" },
+    ]);
+    const { openDiscuss } = await start();
+    await openDiscuss(teamId);
+    await contains(".o-mail-DiscussCategoryItem", { text: "general" });
+    await contains(".o-mail-DiscussCategoryItem.o-active", { text: "team" });
+    await click("[aria-label='Messages']");
```

Comment:
have at least a node or a class specified, this will probably have to check all the DOM nodes to find the matching attributes